reconcile/utils/openshift_resource.py

```diff
@@ -226,6 +226,9 @@ def has_qontract_annotations(self):
 
         return True
 
+    def has_owner_reference(self):
+        return len(self.body['metadata'].get('ownerReferences', [])) > 0
```

Comment:
Simple nit:
```suggestion
        return bool(self.body['metadata'].get('ownerReferences', []))
```
which is closer to English' "the metadata has owner deferences".

Reply:
makes sense. i really was going towards the bool direction. thanks!